README.md

```diff
@@ -60,14 +60,14 @@ If it doesn't, you can manually point your browser to http://localhost:3000.
 
 ## A quick tour
 
-You can log into the app by providing a user name, say `Alice`. For simplicity
+You can log into the app by providing a user name, say `Alice`. For the simplicity
```

Comment:
You can probably submit this PR to the original repo...

Reply:
![image](https://user-images.githubusercontent.com/61805754/129052235-38a3ef96-7f45-40b9-afa6-93ec06905ddd.png)